### PR TITLE
Test if CardFormComponent is present in view

### LINKED ISSUE #431: write unit tests for card generation
Jest test
## acceptance criteria
- [x] Test to check if all components are visible in view

### PR DESCRIPTION
**Pull request by Te4-Jacob-sandstrom**
## ⚙️ What issue did you implement or fix?
Fixes #431

## 📜 Summary
- tests if AdminCardView has component: CardFormComponent.vue

## 📝 Type of change

Please delete options that are not relevant.

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 This change requires a documentation update

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## 💩 How can this implementation be improved?
- Change bar into zar. :shipit:

### CODE REVIEW

**Comment by itggot-linus-styren** [comment]
To make integration tests pass, please cherry-pick commits from #434 (23dcfd71bf646111fed2d97c7cc15dedcfbbc183 and 9003ce0cb508cd220c9262a4bf184e3ece6daf68)
```
git fetch --all
git cherry-pick 23dcfd7
git cherry-pick 9003ce0
git push
```

File: frontend/src/views/__tests__/AdminCardView.spec.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { shallowFactory } from '@/helpers/testFactoryHelpers';
+import { Wrapper } from '@vue/test-utils';
+import AdminCardsView from '../AdminCardsView.vue';
+
+describe('AdminCardsView.vue', () => {
+  
+  it('CardFormComponent exists in view', () => {
+    const wrapper: Wrapper<AdminCardsView> = shallowFactory(AdminCardsView);
+    expect(wrapper.find({name: 'CardFormComponent'}).exists()).toBe(true);
+
```
Comment (te4-adrian-almetun-smeds) [inline frontend/src/views/__tests__/AdminCardView.spec.ts:12]:
```suggestion
```

File: frontend/src/views/__tests__/AdminCardView.spec.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
```
Comment (te4-adrian-almetun-smeds) [inline frontend/src/views/__tests__/AdminCardView.spec.ts:2]:
```suggestion
```

**Review by te4-adrian-almetun-smeds — CHANGES_REQUESTED** [review]
Other than having some excessive whitespace this looks good. Even basic tests are tests we need :+1:

**Review by te4-david-jensen — APPROVED** [review]
Looks good! 👍 :shipit:

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
Looks good :+1: :shipit:

**Pull request merged**